pkg/label/label.go

```diff
@@ -62,3 +75,19 @@ func IsNydusMetaLayer(labels map[string]string) bool {
 	_, ok := labels[NydusMetaLayer]
 	return ok
 }
+
+func IsTarfsDataLayer(labels map[string]string) bool {
+	if labels == nil {
+		return false
```

Comment:
Indexing nil map is safe, we don't have to check if it is nil

Reply:
Fixed